src/rt/cover.c

```diff
@@ -145,25 +145,68 @@ static inline void cover_toggle_check_0_1_u_z(uint8_t old, uint8_t new,
 #define COVER_TGL_SIGNAL_DETAILS(signal, size)
 #endif
 
-// TODO: Could multi-bit signals be optimized with vector instructions ?
-#define DEFINE_COVER_TOGGLE_CB(name, check_fnc)                               \
-   static void name(uint64_t now, rt_signal_t *s, rt_watch_t *w, void *user)  \
-   {                                                                          \
-      uint32_t s_size = s->shared.size;                                       \
-      rt_model_t *m = get_model();                                            \
-      const int32_t tag = (uintptr_t)user;                                    \
-      int32_t *toggle_01 = get_cover_counter(m, tag);                         \
-      int32_t *toggle_10 = toggle_01 + 1;                                     \
-      COVER_TGL_CB_MSG(s)                                                     \
-      for (int i = 0; i < s_size; i++) {                                      \
-         uint8_t new = ((uint8_t*)signal_value(s))[i];                        \
-         uint8_t old = ((uint8_t*)signal_last_value(s))[i];                   \
-         check_fnc(old, new, toggle_01, toggle_10);                           \
-         toggle_01 += 2;                                                      \
-         toggle_10 += 2;                                                      \
-      }                                                                       \
-      COVER_TGL_SIGNAL_DETAILS(s, s_size)                                     \
-   }                                                                          \
+// Set to match word size of common PC and optimize for int64_t comparison
+#define TOGGLE_BATCH_SIZE 8
```

Comment:
Use `sizeof(uint64_t)` instead of this. It's shorter and a `#define` might lead someone to wrongly assume that it can be changed when the only possible value is 8.